JKH/Day3/자릿수 더하기.js

```diff
@@ -0,0 +1,18 @@
+function solution(n)
+{
+  let pow = 0;
+  let answer = 0;
+
+  while(1) {
+    if (n / Math.pow(10, pow) < 1) break;
+    pow++;
+  }
```

Comment:
```suggestion
  let pow = String(n).length;
  let answer = 0;
```
이렇게 하면 더 간단하게 N이 몇 자릿수인지 알아낼 수 있을거 같습니다!